### ISSUE TITLE
Missing DDL Auto Completion in Ecplise editor 

### ISSUE THREAD

**Issue by arunvitto** [dbeaver/dbeaver#16866]
In Eclipse Editor SQL editor window does not auto-complete the DDL queries like create, alter, etc for MySql

**Comment by LonwoLonwo** [comment]
Hello @arunvitto 

Please Provide:

#### System information: 
- Operating system (distribution) and version
- DBeaver version
- Additional extensions

And cases of this issue with examples. Screens will be very helpful.

**Comment by E1izabeth** [comment]
Did you install DBeaver as an Eclipse plugin or a standalone application?
If you installed DBeaver as a plugin, please, provide information about Eclipse version and other installed plugins versions.

**Comment by arunvitto** [comment]
> Hello @arunvitto
> 
> Please Provide:
> 
> #### System information:
> * Operating system (distribution) and version
> * DBeaver version
> * Additional extensions
> 
> And cases of this issue with examples. Screens will be very helpful.

### **Operating System**
<pre>NAME=&quot;Ubuntu&quot;
VERSION=&quot;20.04.4 LTS (Focal Fossa)&quot;
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME=&quot;Ubuntu 20.04.4 LTS&quot;
VERSION_ID=&quot;20.04&quot;
</pre>

### **DBeaver version**
<pre> VERSION = 22.0.5
</pre>

### **Additional extensions**
<pre>

1. buildship-gradle-integration
2. darkest-dark-theme-devstyle
3. eclemma-java-code-coverage
4. eclipse-enterprise-java-and-web-developer-tools
5. eclipse-m2e-maven-support-eclipse-ide
6. eclipse-marketplace-client
7. eclipse-web-developer-tools-0/promo
8. egit-git-integration-eclipse
9. subclipse
10. wild-web-developer-html-css-javascript-typescript-nodejs-angular-json-yaml-kubernetes-xml
</pre>

### **Screen Shots**

![image](https://user-images.githubusercontent.com/36920562/175284289-293364f7-7ee7-449e-a020-b925669fc11c.png)

**Comment by arunvitto** [comment]
> Did you install DBeaver as an Eclipse plugin or a standalone application? If you installed DBeaver as a plugin, please, provide information about Eclipse version and other installed plugins versions.

Yes Installed as an Eclipse plugin

Pl. refer to the below link

[https://github.com/dbeaver/dbeaver/issues/16866#issuecomment-1164271778](url)

**Comment by LonwoLonwo** [comment]
For now DBeaver doesn't have auto-completion proposals for CREATE, ALTER statements.

Please check auto-complete for statements like 

`SELECT * FROM <catalog_name>.`

![2022-06-23 15_04_12-DBeaver Ultimate 22 1 0 - _MariaDB 10 1_Script-169](https://user-images.githubusercontent.com/45152336/175294500-e262ec81-7342-4362-9cc2-6933a5fccbf7.png)

Is it working in this case?

**Comment by arunvitto** [comment]
> For now DBeaver doesn't have auto-completion proposals for CREATE, ALTER statements.
> 
> Please check auto-complete for statements like
> 
> `SELECT * FROM <catalog_name>.`
> 
> ![2022-06-23 15_04_12-DBeaver Ultimate 22 1 0 - _MariaDB 10 1_Script-169](https://userimages.githubusercontent.com/45152336/175294500-e262ec81-7342-4362-9cc2-6933a5fccbf7.png)
> 
> Is it working in this case?

Yes Select Queries are working fine

**Comment by LonwoLonwo** [comment]
Ok, thanks @arunvitto 

Do you expect keywords like "TABLE", "VIEW" etc. after CREATE/ALTER keywords?

**Comment by arunvitto** [comment]
> Ok, thanks @arunvitto
> 
> Do you expect keywords like "TABLE", "VIEW" etc. after CREATE/ALTER keywords?

Yes...  Lots of keywords required like TABLE, VIEW, Auto_increment, primary key, not null, and so on
